wgs/da/README.md

```diff
@@ -4,4 +4,6 @@
 
 |  №  |      Date       | Agenda | Notes | Recording |
 |:---:|:---------------:|:------:|:-----:|:---------:|
-| 1 | Date | Agenda Link | Notes Link | Recording Link |
+| 1 | 6 December 2023 | Agenda is in Meeting Notes | [Meeting Notes](https://github.com/celestiaorg/data-availability-wg/blob/main/meeting-notes/2023-12-06-initial-meeting.md) | N/A |
+| 2 | 22 January 2024 | Agenda is in Meeting Notes | [Meeting Notes](https://github.com/celestiaorg/data-availability-wg/blob/main/meeting-notes/2024-01-22-meeting-2.md) | N/A |
```

Comment:
will coordinate with @ramin to migrate these in the CIP repo itself, and use the template as an example template for WGs

Reply:
i'm going to close this so Ramin can open a PR to maintain authorship of the documentation for CIP DA WG